core/Plugin/Dimension/VisitDimension.php

```diff
@@ -319,20 +319,53 @@ public static function getAllDimensions()
 
     /**
      * @ignore
+     * @param VisitDimension[] $dimensions
      */
-    public static function sortByRequiredFields($a, $b)
+    public static function sortDimensions($dimensions)
```

Comment:
The sort result for PHP7 was different but this way I actually discovered a bug that we did not resolve required fields correctly